Utilities/dronekit_wrappers.py

```diff
@@ -0,0 +1,93 @@
+# Standard Library
+import coloredlogs
+import dronekit
+import logging
+from math import radians, sin, cos
+from pymavlink import mavutil
+import threading
+from sys import stdout
+from time import sleep, time
+
+# Ours
+import constants as c
+
+def get_velocity_message(message_factory, (velocity_x, velocity_y, velocity_z)):
+    """
+    Returns a DroneKit object that represents a mavlink message which moves
+    the drone at a certain velocity.
+
+    Parameters
+    ----------
+    message_factory: dronekit.vehicle.message_factory
+        Used to create the message
+    (velocity_x, velocity_y, velocity_z): (Double, Double ,Double)
+        The vector to travel along
+
+    Returns:
+    ----------
+    MAVLink_message (a DroneKit object)
+    """
+    return message_factory.set_position_target_global_int_encode(
+            0,       # time_boot_ms (not used)
+            0, 0,    # target system, target component
+            mavutil.mavlink.MAV_FRAME_GLOBAL_RELATIVE_ALT_INT, # frame
+            0b0000111111000111, # type_mask (only speeds enabled)
```

Comment:
Consider creating an enum for these bitmasks

Reply:
Could you elaborate on this suggestion more? I'm not sure what you mean.